src/Schema.php

```diff
@@ -90,6 +93,14 @@ public static function ensureFrontControllerAccess() {
     static::createOrPrependFile(ABSPATH . '.htaccess', $template, "\n");
   }
 
+  /**
+   * Set Cache-Control header with a TTL of 1 year for all assets.
```

Comment:
It's perfectly okay to repeat the inline comment from init as PHPDoc summary line if there's nothing further to say.
```suggestion
   * Force a long client-side caching time for assets with "ver" query string.
```